DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt

```diff
@@ -95,12 +95,10 @@ fun addToCollection(info: DittoHeartbeatInfo, config: DittoHeartbeatConfig, ditt
         "metaData" to metaData,
```

Comment:
I know this isn't changing as part of this PR, but I just noticed this is capitalized `metaData` here and `metadata` on the Swift version - we probably want to align those.

Reply:
Updated with this commit: https://github.com/getditto/DittoSwiftTools/pull/98/commits/8921870cc94bf6cc7901fedeacbd27d132cb99fb